scripts/check-repo-guard.ps1

```
@@ -0,0 +1,313 @@
+        UsesCapture = $true
+    }
+)
+
+function Test-LineForSecrets {
+    param(
+        [Parameter(Mandatory = $true)]
+        [AllowEmptyString()]
+        [string]$Line
+    )
+
+    $matches = @()
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'matches' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)